picard/ui/infodialog.py

```diff
@@ -297,7 +297,7 @@ def _display_info_tab(self):
         if album.errors:
             tabWidget.setTabText(tab_index, _("&Errors"))
             text = '<br />'.join(map(lambda s: '<font color="darkred">%s</font>' %
-                                     '<br />'.join(unicode(QtCore.Qt.escape(s))
+                                     '<br />'.join(str(cgi.escape(s))
```

Comment:
https://wiki.qt.io/Transition_from_Qt_4.x_to_Qt5 says `Qt.escape()` is deprecated in favor of `QString.toHtmlEscaped()`, `cgi.escape()` looks weird here, and it doesn't escape `"` by default.

There is also Py3 `html.escape()` https://docs.python.org/3/library/html.html 
https://wiki.python.org/moin/EscapingHtml

Not sure which one is better.